src/components/Article/Article.jsx

```diff
@@ -1,6 +1,66 @@
-// import the React library here
-// import a css file containig article styles
+import React from 'react';
+import './Article.css';
 
-// Create an Article function returning the HTML of article block
+const Article = () => (
+  <article className="article">
+    <h1 className="article__title">Headline</h1>
+    <p className="article__paragraph">
```

Comment:
we put empty line between closed & open tag
use it in all cases
```suggestion
    <h1 className="article__title">Headline</h1>
    
    <p className="article__paragraph">
```